x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/pick_fields_step/components/bucket_span_estimator/bucket_span_estimator.tsx

```diff
@@ -15,14 +16,23 @@ interface Props {
 }
 
 export const BucketSpanEstimator: FC<Props> = ({ setEstimating }) => {
+  const { jobCreator, jobCreatorUpdate } = useContext(JobCreatorContext);
   const { status, estimateBucketSpan } = useEstimateBucketSpan();
+  const [noDetectors, setNoDetectors] = useState(jobCreator.detectors.length === 0);
 
   useEffect(() => {
     setEstimating(status === ESTIMATE_STATUS.RUNNING);
   }, [status]);
 
+  useEffect(() => {
+    setNoDetectors(jobCreator.detectors.length === 0);
+  }, [jobCreatorUpdate]);
+
   return (
-    <EuiButton disabled={status === ESTIMATE_STATUS.RUNNING} onClick={estimateBucketSpan}>
+    <EuiButton
```

Comment:
The estimate bucket span is returning an error for me - looks like the time range has defaulted to last 15 mins in the URL.

![image](https://user-images.githubusercontent.com/7405507/66560143-a9304480-eb4e-11e9-8350-c406413f128b.png)

Reply:
The full time range for the index should probably be used here. There is a card in the project to  investigate this.
https://github.com/elastic/kibana/projects/30#card-27509086